variants/STM32WBxx/WB55R(C-E-G)V/variant_STEAM32_WB55RG.h

```diff
@@ -0,0 +1,235 @@
+#define GPIO4 47 // P25
+
+// 4 directions switches
+#define GPIO5 48
+#define GPIO6 49
+#define GPIO7 50
+#define GPIO8 51
+
+// STeaMi's PinName
+#define ACTIVATE_3V3_STLINK PB2
+#define CS_DISPLAY          PD0
+#define I2C_EXT_SCL         PC0
+#define I2C_EXT_SDA         PC1
+#define I2C_INT_SCL         PB8
+#define I2C_INT_SDA         PB9
+#define INT_ACC             PC13
+#define INT_DIST            PB12
+#define INT_EXPANDER        PB0
+#define INT_MAG             PD1
+#define JACDAC_DATA_TX      PB6
+#define JACDAC_DATA_RX      PB7
+#define MENU_BUTTON         PA0
+#define MIC_CLK             PA3
+#define MIC_IN              PA10
+
+#define P0  PC4
+#define P1  PA5
+#define P2  PC5
+#define P3  PA2
+#define P4  PA4
+#define P5  PA7
+#define P6  PC3
+#define P7  PA9
+#define P8  PA15
+#define P9  PC2
+#define P10 PA6
+#define P11 PA8
+#define P12 PC6
+#define P13 PB13
+#define P14 PB14
+#define P15 PB15
+#define P16 PE4
+#define P19 PC1
+#define P20 PC0
+#define P24 GPIO3
+#define P25 GPIO4
+#define P28 GPIO1
+#define P29 GPIO2
+
+#define RST_DISPLAY  PA12
+#define RST_EXPANDER PB1
+
+#define SERIAL_RX PB10
+#define SERIAL_TX PB11
+
+#define SPEAKER PA11
+
+#define SPI_EXT_MISO PB14
+#define SPI_EXT_MOSI PB15
+#define SPI_EXT_SCK  PB13
+
+#define SPI_INT_MISO PB4
+#define SPI_INT_MOSI PB5
+#define SPI_INT_SCK  PA1
+
+// On-board LED pin number
+#define LED1 PC10
+#define LED2 PC11
+#define LED3 PC12
+#define LED4 PH3
+
+#ifndef LED_BUILTIN
+  #define LED_BUILTIN LED1
+#endif
+
+#define LED_BLUE  LED1
+#define LED_GREEN LED2
+#define LED_RED   LED3
+
+#define LED_BLE LED4
+
+// On-board user button
+#define A_BUTTON     PA7
+#define B_BUTTON     PA8
+#define DOWN_BUTTON  GPIO6
+#define LEFT_BUTTON  GPIO7
+#define MENU_BUTTON  PA0
+#define RIGHT_BUTTON GPIO5
+#define UP_BUTTON    GPIO8
+
+#ifndef USER_BTN
+  #define USER_BTN MENU_BUTTON
+#endif
+
+// SPI Definitions
+#define PIN_SPI_SS   P16
+#define PIN_SPI_MOSI SPI_EXT_MOSI
+#define PIN_SPI_MISO SPI_EXT_MISO
+#define PIN_SPI_SCK  SPI_EXT_SCK
+
+// I2C Definitions
+#define PIN_WIRE_SDA I2C_EXT_SDA
+#define PIN_WIRE_SCL I2C_EXT_SCL
+
+// Timer Definitions
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#ifndef TIMER_TONE
+  #define TIMER_TONE TIM16
+#endif
+#ifndef TIMER_SERVO
+  #define TIMER_SERVO TIM17
+#endif
+
+// UART Definitions
+#ifndef SERIAL_UART_INSTANCE
+  #define SERIAL_UART_INSTANCE 0  // Connected to ST-Link
+#endif
+
+// Default pin used for 'Serial' instance (ex: ST-Link)
+// Mandatory for Firmata
+#ifndef PIN_SERIAL_RX
+  #define PIN_SERIAL_RX SERIAL_RX
+#endif
+#ifndef PIN_SERIAL_TX
+  #define PIN_SERIAL_TX SERIAL_TX
+#endif
+
+// Only 512k provided for cpu1, so defined the FLASH_PAGE_NUMBER
+// for EEPROM emulation to the last 512k pages.
+#define FLASH_PAGE_NUMBER 127
+
+// Extra HAL modules
+#if !defined(HAL_QSPI_MODULE_DISABLED)
+  #define HAL_QSPI_MODULE_ENABLED
+#endif
+
+/* backward compatibility */
+#define ARDUINO_PNUCLEO_WB55RG
```

Comment:
Used for?

Reply:
Come from [here](https://github.com/stm32duino/Arduino_Core_STM32/blob/ea74156acd823b6d14739f389e6cdc648f8ee36e/variants/STM32WBxx/WB55R(C-E-G)V/variant_P_NUCLEO_WB55RG.h#L135). I will remove it